src/components/AlertBox/index.tsx

```diff
@@ -23,10 +31,19 @@ export const AlertBox: FC<AlertBoxProps> = ({
   withIcon = true,
   ...restProps
 }) => {
+  const { colorMode } = useColorMode()
+
+  const isDarkMode = colorMode === "dark"
+
   if (status === "magic") {
     return (
-      <Alert status="info" {...restProps} backgroundColor="brand.100">
-        {withIcon && <AlertIcon color="brand.500" />}
+      <Alert
+        status="info"
+        {...restProps}
+        backgroundColor={isDarkMode ? "rgba(255, 255, 255, 0.08)" : "brand.100"}
+        borderColor={isDarkMode ? "brand.400" : "brand.500"}
+      >
+        {withIcon && <Image h="20px" src={magicAlertIcon} mr={2} />}
```

Comment:
Non-blocking:

This icon is the same as if you just write:
`{withIcon && <AlertIcon color="<proper_color>" />}`
but unfortunately in this case the gradient is not working when we use it as `color=gradient.3` 🤔 Not sure why, didn't really dig into it. I think we can also use the icon instead like you did there 👍

Reply:
Yeah unfortunately the gradient colors don't work on the AlertIcon. I think it expects a `colorScheme` prop so that it can change contextually for variants and light/dark mode by pulling from the theme. That's why I went with the image